src/interactive/tap/tap-in-mini-chart.js

```diff
@@ -0,0 +1,49 @@
+import KEYS from '../../constants/keys';
+import NUMBERS from '../../constants/numbers';
+
```

Comment:
Everything in the `interactive` folder is just a copy / paste of the generic `interactive` module from La-Vie. This will be removed when that has been refactored and place somewhere else. Scatter-plot specific interactions should be placed in `picasso-definition/interactions`.